src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -28,7 +28,12 @@ class VideoInterface(BaseDataInterface):
     # Other suffixes, while they can be opened by OpenCV, are not supported by DANDI so should probably not list here
     info = "Interface for handling standard video file formats."
 
-    def __init__(self, file_paths: list, verbose: bool = False):  # TODO - debug why List[FilePathType] fails
+    def __init__(
+        self,
+        file_paths: list,
+        metadata_key_name: str = "Videos",
+        verbose: bool = False,
+    ):  # TODO - debug why List[FilePathType] fails
```

Comment:
```suggestion
    def __init__(
        self,
        file_paths: list,
        verbose: bool = False,
        *,
        metadata_key_name: str = "Videos",
    ):
```

This is what he means

Reply:
Thanks Cody, way better to make a concrete suggestion than pasting a wall of text x D

Reply:
I see, but do we gain anything from doing that, here?
I'm fine with either option

Reply:
It forces any new usage of the new argument to use key/val pair instead of allow the possibility of position args

The bigger gain is doing everywhere, but that's a big thing that we would need to warn people about for a long time beforehand